### PR TITLE
Double dqn update

### PR DESCRIPTION
**Pull request by lgraesser**

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -15,6 +15,7 @@ dependencies:
     - pip install scipy
     - pip install matplotlib
     - pip install pandas
+    - pip install hyperopt
```
Comment (eligiblekeng) [inline circle.yml]: kill this dipshit

File: requirements.txt [hunk 2]
```diff
@@ -5,6 +5,7 @@ scipy==0.18.0
 matplotlib==1.4.3
 pandas==0.18.1
 pytest-cov==2.4.0
+hyperopt==0.1
```
Comment (eligiblekeng) [inline requirements.txt]: kill him too

File: circle.yml (hunk 1 above)
Comment (lgraesser) [inline circle.yml]: Haha. But hyperoptimizer still imports it so the CI breaks without it.

File: requirements.txt (hunk 2 above)
Comment (lgraesser) [inline requirements.txt]: Same here

File: circle.yml (hunk 1 above)
Comment (kengz) [inline circle.yml]: oh shoot you're right

Comment (lgraesser) [inline circle.yml]: gonna merge. Then we can take hyperopt out from master in one go?

Comment (kengz) [inline circle.yml]: yeah gonna do that when during the optimizer work

**Pull request merged**